hudi-common/src/main/java/org/apache/hudi/common/table/log/HoodieUnMergedLogRecordScanner.java

```diff
@@ -19,6 +19,7 @@
 package org.apache.hudi.common.table.log;
 
 import org.apache.hudi.common.model.DeleteRecord;
+import org.apache.hudi.common.model.HoodieKey;
```

Comment:
The changes in this file should be gone after rebase.

Reply:
then, can we do the rebase?

Reply:
yeah. Will do.

Reply:
I rebased, but it did not go. I think it is from your branch. I will try to override it.